web3/middleware/pythonic.py

```diff
@@ -360,5 +364,10 @@ def to_hexbytes(num_bytes, val, variable_length=False):
         'evm_snapshot': hex_to_integer,
         # Net
         'net_peerCount': to_integer_if_hex,
+        # Parity
```

Comment:
@carver and @dylanjw what do you think about isolating these into their own parity specific middleware.

Reply:
Yeah, that seems like the right way to go. We could extend `Parity.attach(w3)` to insert the parity middleware for you.